docs/source/getting-started/first-feature.rst

```diff
@@ -78,6 +78,11 @@ Open a new terminal and run the following commands::
 
 The result is an up-to-date TypeScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
 
+To make sure that Yarn picks up the newly generated JavaScript code,
```

Comment:
For the sake of consistency with the rest of the text:
```suggestion
To make sure that yarn picks up the newly generated JavaScript code,
```

Reply:
Yarn capitalizes itself on its own web page so I think we should be consistent with that.